apps/staking/app/mystakes/modules/BalanceModule.tsx

```diff
@@ -39,12 +39,14 @@ function useTotalStakedAmount(params?: { addressOverride?: Address }) {
     [params?.addressOverride, connectedAddress]
   );
 
+  const enabled = !!address;
```

Comment:
No worried about need to set this on mount or anything? e.g. using a useState and useEffect or something?

Reply:
I forget is there is any magic in react 19 where you don't need to worry about this 🤔

Reply:
Nah its not needed at all because its just a const boolean calculation, don't need to memoize it